### ISSUE TITLE
Wrong page break

### ISSUE THREAD

**Issue by rodolfo-medina** [gsilano/EuropeCV#39]
Hi: in the definition of  \section there must be something wrong, because it happens to me that the page sometimes breaks just after the horizontal rule that starts any new section, between the horizontal rule and the following text: that it shoudn't absolutely do.

Thanks,

Rodolfo

**Comment by gsilano** [comment]
Hi! Could you attach your code so that I can test it? Thanks!

**Comment by rodolfo-medina** [comment]
Yes: this is my cv and the problem occurs on the end of pp.6 and 7.  The problem seems to be solved if, in the definition of section, I remove '\nopagebreak'.  I process the tex file with 'pdflatex europass-cv.tex'.  Thank you.  Github won't allow to me to upload the extension `.tex', so I'm attaching it as a '.txt' file, but it's actually a tex file.


[europass-cv.txt](https://github.com/user-attachments/files/16726856/europass-cv.txt)

[europass-cv.pdf](https://github.com/user-attachments/files/16726809/europass-cv.pdf)

**Comment by gsilano** [comment]
The issue is caused by the beginning of the "Cinema Indipendente" section. A simple way to fix it is to add a "\vspace*{0.5em}" command. Feel free to adjust the argument (0.5em) according to your preferences. Alternatively, you could force a new page with the "\newpage" command. 

Since this problem is specific to your CV and does not affect any others, I won't implement these changes in the repository on CTAN.

If you need any further assistance, please don't hesitate to ask.

P.S. Next time, please share all files in a zip folder, as some images and other minor elements were missing during compilation.
